serokell-style.md

```diff
@@ -288,372 +300,252 @@ Prefer two-line imports for such standard containers.
 
 ```haskell
 import Data.Map (Map)
-
-import qualified Data.Map as Map hiding (Map)
+import qualified Data.Map as Map
 ```
 
-Such tools as `stylish-haskell` can make your import section look very nice!
-
 ### Data Declarations
 
-Align the constructors in a data type definition.  Example:
+Align the constructors in a data type definition. Examples:
 
 ```haskell
-data Tree a = Branch !a !(Tree a) !(Tree a)
-            | Leaf
-```
-
-For long type names the following formatting is also acceptable:
+data Tree a
+  = Branch a (Tree a) (Tree a)
+  | Leaf
 
-```haskell
 data HttpException
-    = InvalidStatusCode Int
-    | MissingContentHeader
+  = InvalidStatusCode Int
+  | MissingContentHeader
 ```
 
 Format records as follows:
 
 ```haskell
 data Person = Person
-    { firstName :: !String  -- ^ First name
-    , lastName  :: !String  -- ^ Last name
-    , age       :: !Int     -- ^ Age
-    } deriving (Eq, Show)
-```
-
-If you have record with multiple constructors (which is generally bad idea
-because you getters become partial functions but okay if you use
-`-XRecordWildCards`) then align curly braces with
-shift to the constructor name but } should go in the end of last field.
-
-```haskell
-data Address
-    = PubKeyAddress
-        { addrKeyHash :: !(AddressHash PublicKey) }
-    | ScriptAddress
-        { addrScriptHash   :: !(AddressHash Script)
-        , addrDistribution :: ![(AddressHash PublicKey, Coin)] }
-    deriving (Show, Eq)
-```
-
-If there is only one field for every constructor more compact style is allowed.
-
-```haskell
-data Address
-    = PubKeyAddress { addrKeyHash    :: !(AddressHash PublicKey) }
-    | ScriptAddress { addrScriptHash :: !(AddressHash Script)    }
-    deriving (Show, Eq)
+  { firstName :: String  -- ^ First name
+  , lastName  :: String  -- ^ Last name
+  , age       :: Int     -- ^ Age
+  } deriving (Eq, Show)
 ```
 
-Type classes in `deriving` section should be always surrounded by
-parentheses. Space between names is optional.
+You _must_ avoid having records with multiple constructors because
```

Comment:
> _must_ avoid

It's the same as "you _must_ try". Do, or do not :wink:

Reply:
The intended meaning is "you must never use them". I will update the wording to make it clearer.